diff --git a/node_modules/toastify-react-native/.babelrc b/node_modules/toastify-react-native/.babelrc
old mode 100644
new mode 100755
diff --git a/node_modules/toastify-react-native/.expo/README.md b/node_modules/toastify-react-native/.expo/README.md
old mode 100644
new mode 100755
diff --git a/node_modules/toastify-react-native/.expo/settings.json b/node_modules/toastify-react-native/.expo/settings.json
old mode 100644
new mode 100755
diff --git a/node_modules/toastify-react-native/CONTRIBUTING.md b/node_modules/toastify-react-native/CONTRIBUTING.md
old mode 100644
new mode 100755
diff --git a/node_modules/toastify-react-native/LICENSE b/node_modules/toastify-react-native/LICENSE
old mode 100644
new mode 100755
diff --git a/node_modules/toastify-react-native/README.md b/node_modules/toastify-react-native/README.md
old mode 100644
new mode 100755
diff --git a/node_modules/toastify-react-native/components/ToastManager.tsx b/node_modules/toastify-react-native/components/ToastManager.tsx
old mode 100644
new mode 100755
index 698d8e2..2720894
--- a/node_modules/toastify-react-native/components/ToastManager.tsx
+++ b/node_modules/toastify-react-native/components/ToastManager.tsx
@@ -49,7 +49,7 @@ class ToastManager extends Component<ToastManagerProps, ToastManagerState> {
   }
 
   static info = (text: string, position: string) => {
-    ToastManager.__singletonRef?.show(text, Colors.info, 'ios-information-circle', position)
+    ToastManager.__singletonRef?.show(text, Colors.info, 'information-circle', position)
   }
 
   static success = (text: string, position?: string) => {
@@ -193,7 +193,7 @@ class ToastManager extends Component<ToastManagerProps, ToastManagerState> {
           ]}
         >
           <TouchableOpacity onPress={this.hideToast} activeOpacity={0.9} style={styles.hideButton}>
-            <Icon name='ios-close-outline' size={22} color={Colors[theme].text} />
+            <Icon name='close-outline' size={22} color={Colors[theme].text} />
           </TouchableOpacity>
           <View style={styles.content}>
             <Icon name={icon} size={22} color={barColor} style={styles.iconWrapper} />
diff --git a/node_modules/toastify-react-native/components/styles.ts b/node_modules/toastify-react-native/components/styles.ts
old mode 100644
new mode 100755
diff --git a/node_modules/toastify-react-native/config/theme.ts b/node_modules/toastify-react-native/config/theme.ts
old mode 100644
new mode 100755
diff --git a/node_modules/toastify-react-native/index.ts b/node_modules/toastify-react-native/index.ts
old mode 100644
new mode 100755
diff --git a/node_modules/toastify-react-native/utils/defaultProps.ts b/node_modules/toastify-react-native/utils/defaultProps.ts
old mode 100644
new mode 100755
diff --git a/node_modules/toastify-react-native/utils/interfaces.ts b/node_modules/toastify-react-native/utils/interfaces.ts
old mode 100644
new mode 100755
